### ISSUE TITLE
Error when trying to use this in WebPack.

### ISSUE THREAD

**Issue by chr15m** [feross/timidity#8]
Hi @feross - I know you're very busy so hopefully this won't take much of your time.

When I try to do a simple `import * as timidity from "timidity";` and then run that file through webpack I get the following 2 errors:

```
ERROR in ./node_modules/timidity/libtimidity.wasm
Module not found: Error: Can't resolve 'env' in '/home/chrism/dev/something/node_modules/timidity'
 @ ./node_modules/timidity/libtimidity.wasm
 @ ./node_modules/timidity/index.js
 @ ./src/js/index.js

ERROR in ./node_modules/timidity/libtimidity.wasm
WebAssembly module is included in initial chunk.
This is not allowed, because WebAssembly download and compilation must happen asynchronous.
Add an async splitpoint (i. e. import()) somewhere between your entrypoint and the WebAssembly module:
* ./src/js/index.js --> ./node_modules/timidity/index.js --> ./node_modules/timidity/libtimidity.wasm
```

I'm not a heavy webpack user but maybe you have some pointers as to how I can fix this? Will issue a PR if I get it working.

**Comment by chr15m** [comment]
If I follow the advice in the second error it goes away but I still get the first error.

**Comment by chr15m** [comment]
Just saw this: https://github.com/webpack/webpack/issues/7352

I will follow up with the workaround linked there.

**Comment by chr15m** [comment]
I got webpack to build without errors using the suggested workaround:

```
  "module": {
    "rules": [
      {
        "test": /libtimidity\.wasm$/,
        "type": "javascript/auto", // ← !!
        "loader": "file-loader",
        "options": {
          "publicPath": "/"
        }
      }
    ]
  }
```

However I couldn't figure out how to use that build. In the end I gave up and just used [bg-sound](https://github.com/feross/bg-sound) in a script tag and banged a `<bg-audio src="...mid">` into the page.

Hopefully in future somebody will get a demo app working with Timidity using Webpack.
